Build failure: appimageupdate

### Nixpkgs version

- Unstable (25.11)

### Steps to reproduce

nix-build -A appimageupdate

### Can Hydra reproduce this build failure?

Yes, Hydra can reproduce this build failure.

### Link to Hydra build job

https://hydra.nixos.org/build/310523017

### Relevant log output

```console
> CMake Error at CMakeLists.txt:1 (cmake_minimum_required):
       >   Compatibility with CMake < 3.5 has been removed from CMake.
       >
       >   Update the VERSION argument <min> value.  Or, use the <min>...<max> syntax
       >   to tell CMake that the project requires at least <min> but has been updated
       >   to work with policies introduced by <max> or earlier.
       >
       >   Or, add -DCMAKE_POLICY_VERSION_MINIMUM=3.5 to try configuring anyway.
       >
       >
       > -- Configuring incomplete, errors occurred!
       For full logs, run:
         nix log /nix/store/hrcwjmsy7jzf2qskxabnhvymcvnr1rqd-appimageupdate-2.0.0-alpha-1-20230526.drv
```

### Additional context

_No response_

### System metadata

 - system: `"x86_64-linux"`
 - host os: `Linux 6.12.54, NixOS, 25.11 (Xantusia), 25.11.20251022.01f116e`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.31.2`
 - channels(root): `"home-manager, nixos"`
 - nixpkgs: `/nix/store/sqilaxdxljyalv530z1qrd1q3nd4lyxq-source`

### Notify maintainers



---

**Note for maintainers:** Please tag this issue in your pull request description. (i.e. `Resolves #ISSUE`.)


### I assert that this issue is relevant for Nixpkgs

- [x] I assert that this is a bug and not a support request.
- [x] I assert that this is not a [duplicate of an existing issue](https://github.com/NixOS/nixpkgs/issues?q=is%3Aopen+is%3Aissue+label%3A%220.kind%3A+build+failure%22). 
- [x] I assert that I have read the [NixOS Code of Conduct](https://github.com/NixOS/.github/blob/master/CODE_OF_CONDUCT.md) and agree to abide by it.

### Is this issue important to you?

Add a :+1: [reaction] to [issues you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[issues you find important]: https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc

--- a/pkgs/by-name/ap/appimageupdate/package.nix
+++ b/pkgs/by-name/ap/appimageupdate/package.nix
@@ -19,26 +19,17 @@
 
 stdenv.mkDerivation (finalAttrs: {
   pname = "appimageupdate";
-  version = "2.0.0-alpha-1-20230526";
+  version = "2.0.0-alpha-1-20251018";
 
   src = fetchFromGitHub {
     owner = "AppImageCommunity";
     repo = "AppImageUpdate";
     rev = finalAttrs.version;
-    hash = "sha256-b2RqSw0Ksn9OLxQV9+3reBiqrty+Kx9OwV93jlvuPnY=";
+    hash = "sha256-S3MRBTtPc4S6lqvAZpbZFgOVgsX6GpHZ8PkwEtipT1M=";
   };
-
-  patches = [
-    (fetchpatch {
-      name = "include-algorithm-header.patch";
-      url = "https://github.com/AppImageCommunity/AppImageUpdate/commit/5e91de84aba775ba8d3a4771e4f7f06056f9b764.patch";
-      hash = "sha256-RX2HFAlGsEjXona7cL3WdwwiiA0u9CnfvHMC6S0DeLY=";
-    })
-  ];
 
   postPatch = ''
     substituteInPlace CMakeLists.txt \
-      --replace-fail 'VERSION 1-alpha' 'VERSION ${finalAttrs.version}' \
       --replace-fail 'env LC_ALL=C date -u "+%Y-%m-%d %H:%M:%S %Z"' 'bash -c "echo 1970-01-01 00:00:01 UTC"' \
       --replace-fail 'git rev-parse --short HEAD' 'bash -c "echo unknown"' \
       --replace-fail '<local dev build>' '<nixpkgs build>'

